runelite-client/src/main/java/net/runelite/client/plugins/worldmap/WorldMapPlugin.java

```diff
@@ -110,8 +110,11 @@ public class WorldMapPlugin extends Plugin
 		//A size of 17 gives us a buffer when triggering tooltips
 		final int iconBufferSize = 17;
```

Comment:
This code is generally quite old, and these constants aren't the most helpful at explaining why they are the size they are. Vanilla world map icons are 15x15, so this size is chosen to give them an additional 1-pixel buffer for tooltips. Quest icons need to be 25x25 (which is 5 pixels of additional space on each edge) to accommodate the 20x20 images (the standard 15x15 quest icon with 5 pixels on the right and bottom edges) being bottom-right aligned.

I think I'd prefer if we update this whole static block to center around a `final int worldMapIconSize = 15;` and some variables which indicate how much it should be expanded for "standard" icon overrides (fairy travel, nope icon, mining sites, rooftop courses) and for quest icon overrides.

Reply:
I don't know of any reason offhand. I'm not sure anyone has looked into amending the world map scripts to achieve this. Assuming it is possible to achieve the same effect, I agree it would likely work better to take this approach, though that can happen in a separate PR if it's found to be feasible.